src/solvers/smt2/smt2_dec.cpp

```diff
@@ -148,32 +148,32 @@ decision_proceduret::resultt smt2_dect::read_result(std::istream &in)
 
     const auto &parsed = parsed_opt.value();
 
-    if(parsed.id()=="sat")
-      res=resultt::D_SATISFIABLE;
-    else if(parsed.id()=="unsat")
-      res=resultt::D_UNSATISFIABLE;
+    if(parsed.id() == "sat")
+      res = resultt::D_SATISFIABLE;
+    else if(parsed.id() == "unsat" || parsed.id() == "unknown")
+      res = resultt::D_UNSATISFIABLE;
     else if(
       parsed.id().empty() && parsed.get_sub().size() == 1 &&
       parsed.get_sub().front().get_sub().size() == 2)
     {
-      const irept &s0=parsed.get_sub().front().get_sub()[0];
-      const irept &s1=parsed.get_sub().front().get_sub()[1];
+      const irept &s0 = parsed.get_sub().front().get_sub()[0];
+      const irept &s1 = parsed.get_sub().front().get_sub()[1];
```

Comment:
:pick: This formatting commit noise, would need to be cleaned up before PRing into `develop`. I am not going to block this PR based on this issue, for a non-develop quick fix PR.